src/c_api/c_api.cc

```diff
@@ -16,6 +10,12 @@
 #include <string>
 #include <memory>
 
+#include "xgboost/data.h"
+#include "xgboost/learner.h"
+#include "xgboost/c_api.h"
+#include "xgboost/logging.h"
+#include "xgboost/version_config.h"
```

Comment:
Why is this change necessary?

Reply:
@hcho3 Just couldn't help myself when adding `version_config.h` here (we added some consistency around `include ""` and `include <>` when exporting c++ header).  I can revert this as unnecessary change.